servant-js/examples/counter.hs

```diff
@@ -43,7 +43,7 @@ type TestApi = "counter" :> Post '[JSON] Counter -- endpoint for increasing the
           :<|> "counter" :> Get '[JSON] Counter -- endpoint to get the current value
 
 type TestApi' = TestApi
-           :<|> Raw -- used for serving static files
+          :<|> Raw IO Application -- used for serving static files
```

Comment:
Is `IO` the right `m` for `Raw`? Shouldn't it be `ExceptT ServantErr IO`? It won't matter in these cases, since we don't `enter`, but maybe this is confusing. Which brings up to questions:

- Does leaving it polymorphic work?
- Do we want a type synonym for `Raw (ExceptT ServantErr IO)`?